src/store/elementalChat.js

```diff
@@ -265,9 +274,13 @@ export default {
             }
           }
 
-          const messages = [...result.messages]
+          let messages = [...result.messages]
 
           messages.sort((a, b) => a.createdAt[0] - b.createdAt[0])
+          messages = messages.map((msg) => {
+            msg.createdBy = toUint8Array(msg.createdBy)
+            return msg
+          })
```

Comment:
If this is already mutating the messages I don't you need to use map
```suggestion
          messages = messages.forEach((msg) => {
            msg.createdBy = toUint8Array(msg.createdBy)
          })
```